test/test_main.py

```diff
@@ -70,13 +70,56 @@ def test_create_compare_project(ensure_test1_data):
     project_name = "coclico_test_create_compare_projects"
     metrics_weights = {"mpap0": {"0": 1, "1,2": 2}, "mpap0_test": {"0": 1, "1,2": 2}}
 
-    projects = main.create_compare_project([c1, c2], ref, out, STORE, project_name, metrics_weights)
+    project = main.create_compare_project([c1, c2], ref, out, STORE, project_name, metrics_weights)
+
+    assert project is not None
+    project_json = json.loads(project.to_json())
+    assert len(project_json["jobs"]) > 0  # No empty projects
+    assert project_json["name"].startswith(project_name)
+
+
+def test_create_compare_project_existing_ref(ensure_test1_data):
+    c1 = Path("./data/test1/niv1/")
+    c2 = Path("./data/test1/niv4/")
+    ref = Path("./data/test1/ref/")
+    out = TMP_PATH / "create_compare_project_existing_ref"
+
+    project_name = "test_create_compare_projects_existing_ref"
+    metrics_weights = {"mpap0": {"0": 1, "2": 2}}
+
+    project = main.create_compare_project([c1, c2], ref, out, STORE, project_name, metrics_weights)
+
+    assert np.sum([job.name.startswith("MPAP0_initrinsic_ref") for job in project.jobs]) == 4
```

Comment:
typo:
```suggestion
    assert np.sum([job.name.startswith("MPAP0_intrinsic_ref") for job in project.jobs]) == 4
```
(la typo est aussi dans mpap0 par contre)

Reply:
Pour la typo dans mpap0 : Corrigé dans https://github.com/IGNF/coclico/pull/26 (à compléter après le merge de cette MR